runtime/jcl/exports.cmake

```diff
@@ -366,7 +366,7 @@ omr_add_exports(jclse
 	Java_java_lang_invoke_MethodHandle_invokeExact
 	Java_java_lang_invoke_MethodHandle_requestCustomThunkFromJit
 	Java_java_lang_invoke_MethodHandle_vmRefFieldOffset
-	Java_java_lang_invoke_MethodHandles_00024Lookup_setAccessClassForSecuritCheck
+	Java_java_lang_invoke_MethodHandles_00024Lookup_setAccessClassForSecurityCheck
```

Comment:
This is weird. It appears to be a ghost of https://github.com/Zir0-93/java7-sdk/blob/227e04ea027bd2400ab3e9d2fa3ce5576c6f7239/java7sdk/src/java/lang/invoke/MethodHandles.java
-- There are no other references to this anywhere.

Reply:
This shouldn't be changed because it matches the code. The code can be deleted, but I can do that in a separate PR.

https://github.com/eclipse/openj9/blob/0a3c2988c54c494ecb60dbd5dba45eac3b956f00/runtime/jcl/common/java_dyn_methodhandle.c#L788

Reply:
https://github.com/eclipse/openj9/pull/5956

Reply:
Oops, indeed, https://github.com/eclipse/openj9/commit/cb3bd2ca3b73d994b3b62745fbf057d805937bd7#diff-496bebb71a6246a2673333b9aefbc3c7 (in [spelling-full](https://github.com/jsoref/openj9/commits/spelling-full)) has all the pieces. I just did a bad job of filtering when I was looking at things.